packages/maskbook/src/social-network-adaptor/facebook.com/ui-provider.ts

```diff
@@ -111,7 +114,12 @@ const facebookUI: SocialNetworkUI.Definition = {
             grayscaleAlgorithm: GrayscaleAlgorithm.LUMINANCE,
             password() {
                 // ! Change this might be a breaking change !
-                return currentSelectedIdentity[facebookBase.networkIdentifier].value
+                return (
+                    ProfileIdentifier.getUserName(IdentityProviderFacebook.lastRecognized.value.identifier) ||
```

Comment:
It should be a profile identifier.

Yes:

```
person:twitter.com/differui
```

No:

```
differui
```

Reply:
nice catch